worlds/shorthike/__init__.py

```diff
@@ -88,25 +87,25 @@ def create_regions(self) -> None:
 
         menu_region.connect(main_region)
 
-        if self.options.goal == 0:
+        if self.options.goal == Goal.option_nap:
             # Nap
             self.multiworld.completion_condition[self.player] = lambda state: (state.has("Golden Feather", self.player, get_min_feathers(self, 7, 9))
                 or (state.has("Bucket", self.player) and state.has("Golden Feather", self.player, 7)))
-        elif self.options.goal == 1:
+        elif self.options.goal == Goal.option_photo:
             # Photo
             self.multiworld.completion_condition[self.player] = lambda state: state.has("Golden Feather", self.player, 12)
-        elif self.options.goal == 2:
+        elif self.options.goal == Goal.option_photo:
```

Comment:
2 is Races not Photo
but also you should just be able to do `self.options.goal == "photo"` with the new Options API